blue_badge_app.py

```
@@ -0,0 +1,987 @@
+"""
+
+import tkinter as tk
+from tkinter import ttk, messagebox, filedialog, scrolledtext
+import requests
+import json
+import dns.resolver
+import threading
+import time
+import datetime
+import webbrowser
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)